autometrics/src/lib.rs

```diff
@@ -134,6 +134,46 @@ mod tracker;
 ///
 pub use autometrics_macros::autometrics;
 
+/// # Autometrics custom error labelling
```

Comment:
I wouldn't duplicate the documentation here and in the macros crate. I think it makes sense to have it here, but then I would get rid of it in the other location so that they don't get out of sync.